default_metadata/string/error.catch_all.json

```diff
@@ -0,0 +1,6 @@
+{
+  "_id": "error.catch_all",
+  "_type": "string.error",
+  "description": "A generic validation error not handled by a more concrete validator",
+  "value": "We could not save your answer, please try again"
```

Comment:
I do feel like this message could offer something to users - we couldn't process that file, possibly because of its type - etc, this really needs some input from content/design though so I would ship it as is and bring this to attention for a future tweak

Reply:
Yep, I get what you're saying. Thing is this is a catch-all and generic validator on purpose. If we want to have specific error messages for each of the different possible component errors then a catch-all is not the solution, we have to implement concrete validators, like we have now for virus scan, max size, etc. so we can customise each and all of their error messages.

It's true currently we are only using this for upload errors (the catch all validator checks the component is upload or multiupload) so we could reword the error to talk specifically about file uploads. But the idea was for this to be a fallback, catch-all generic validator with a generic error message instead of blowing up (like now) and giving the "something went wrong" page 😄